packages/terra-badge/CHANGELOG.md

```diff
@@ -2,6 +2,9 @@
 
 ## Unreleased
 
+* Added
+  * Added a11y changes to terra-badge
```

Comment:
This is too vague. Think of how a consumer would understand this. What changed and why?

Here is a good example of changelog entries:
https://github.com/facebook/react/blob/main/CHANGELOG.md